content/en/project/tulpa/index.md

```diff
@@ -0,0 +1,329 @@
+
+<div style="text-align: center">
+   <img src="img/connectome.png" width="600px;" alt=""/>
+</div>
+
+### Deliverable 6: Seed-to-Voxel connectivity results
+I have previously computed a gPPI for this dataset using CONN. I wanted to replicate this task dependent seed to voxel correlation using Nilearn. I have specified the seed based on the group peak activation in the SMA, using the results from the SPM and Randomise analysis. Seed coordination in MNI space are `(-4, 12, 55)`. I have then constructed a sphere around this voxel with a radius of 5 mm. Using this ROI, correlations to all voxels based by task condition were computed. Below is an illustrative example of the connectivity values for one subject and one condition.
+
+<div style="text-align: center">
+   <img src="img/gPPI.png" width="600px;" alt=""/>
+</div>
+
+### Deliverable 7: ML-classifier results
+
+The aim was to train a classifier that can accurately distinguish between different task conditions. I used a majority vote ensemble classifier that combines `LogisticRegression`, `RandomForestClassifier`, and a `SVC`. When classifying the conditions `prep` vs. `write`, the classifier achievs an acciracy of ~80%, so well above chance. However, this is not surprising as the writing conditions will have muhc stronger motor cortex activation and the two conditions are quite different. I then classified the `self`, `tulpa`, and `friend` conditions for preparation and writing respectively. In both conditions, we have an accuracy of ~51% for this 3-group classification problem. Given the three groups, chance levels are at 33.3%, thus an accuracy of 51% is well above chance, YAY!
+
+<br/>
+
+> :information_source: all classifiers were trained and tested on the **connectome**. Next, the plan is to train classifiers on the beta maps of the GLM. See my [to-do list](#My-to-do-list) for further details. 
+
+
+**Prep-write condition**
+
+> Average accuracy = 0.76 <br/>
+> P-value (on 100 permutations): p=0.00
+<div style="text-align: center">
+   <img src="img/ml/ml_p-w_conf.png" width="400px;" alt=""/>
+   <br/>
+   <img src="img/ml/ml_p-w_weights.png" width="400px;" alt=""/>
+   <img src="img/ml/ml_p-w_glass.png" width="400px;" alt=""/>
+</div>
+
+<br/>
+
+**Write condition**
+
+> Average accuracy = 0.52 <br/>
+> P-value (on 100 permutations): p=0.32
+<div style="text-align: center">
+   <img src="img/ml/ml_w_conf.png" width="400px;" alt=""/>
+   <br/>
+   <img src="img/ml/ml_w_weights.png" width="400px;" alt=""/>
+   <img src="img/ml/ml_w_glass.png" width="400px;" alt=""/>
+</div>
+
+<br/>
+
+**Prep condition**
+
+> Average accuracy = 0.37 <br/>
+> P-value (on 100 permutations): p=0.00
```

Comment:
I ran 100 permutation of the lables and none of the permutation yielded a better accuracy in distinguishing the two classes than the true distribution of labels. For a b=publication it would probably be good to run 1000 or more permutations, but with those 100 I ran the stats test revealed a p value of 0.000